examples/elastigroup.yml

```diff
@@ -0,0 +1,61 @@
+SenzaInfo:
+  StackName: hello-world
+  StackVersion: one
+
+SenzaComponents:
+  - AppServerElastigroup:
+      Type: Spotinst::Elastigroup
+      "ServiceToken": "arn:aws:lambda:us-west-2:178579023202:function:spotinst-cloudformation"
```

Comment:
What exactly is this value? Is it a well known static value needed for this to work or just an example?